client/src/fluid/FluidTypes.res

```diff
@@ -127,18 +127,30 @@ module Token = {
      * - the final int is the index of the (pattern -> expr) */
     | TPatternVariable(ID.t, ID.t, string, int)
     | TPatternConstructorName(ID.t, ID.t, string, int)
+
     | TPatternInteger(ID.t, ID.t, int64, int)
     | TPatternString({matchID: ID.t, patternID: ID.t, str: string, branchIdx: int})
+
     | TPatternTrue(ID.t, ID.t, int)
     | TPatternFalse(ID.t, ID.t, int)
-    | TPatternNullToken(ID.t, ID.t, int)
+
+
+    | TPatternNullToken(ID.t, ID.t, int)// why is 'token' in this name
```

Comment:
If needless, I'll remove the "token" bit in a tiny PR

Reply:
Yeah, seems useless